### PR TITLE
changed some config variables of sceewlog. Updated doc and descriptions of this module

### PR DESCRIPTION
**Pull request by billyburgoa**

### CODE REVIEW

File: apps/eew/sceewlog/config/sceewlog.cfg [hunk 1]
```diff
@@ -195,4 +202,4 @@ FCM.activate = false
 # the topic name for Firebase
 # see more about Autho. Key at:https://firebase.google.com/docs/cloud-messaging/auth-server 
 # for topics: https://firebase.google.com/docs/cloud-messaging/android/topic-messaging
-FCM.dataFile = /home/sysop/seiscomp/share/sceewlog/.fcmdatafile
\ No newline at end of file
+FCM.dataFile = /home/sysop/seiscomp/share/sceewlog/.fcmdatafile
```
Comment (FMassin) [inline apps/eew/sceewlog/config/sceewlog.cfg:205]: @USERCONFIGDIR@/.fcmdatafile would be better but maybe in next version?

File: apps/eew/sceewlog/descriptions/sceewlog.xml [hunk 2]
```diff
@@ -151,21 +156,23 @@
...
+			</group>
+			<group name="RegFilters">
+				<parameter name="profiles" type="string" default="global">
 					<description>
-					Path of BNA file that contains closed polygons.
+					Profile names for regionalized alert filter. Use comma for more than one profile.
 					</description>
 				</parameter>
-				<parameter name="profiles" type="string" default="global">
+				<parameter name="bnaFile" type="string" default="@ROOTDIR@/share/sceewlog/closedpolygons.bna">
```
Comment (FMassin) [inline apps/eew/sceewlog/descriptions/sceewlog.xml:166]: It is "none" by default. Please fix it.

File: apps/eew/sceewlog/eews2fcm.py [hunk 3]
```diff
@@ -35,6 +35,7 @@ def __init__(self, fcmDataFile, worldCitiesFile, language = "es-US"):
         self.dic = self.hlalert.csvFile2dic(self.hlalert.dataFile)
         self.language = language
         self.agency = ''
+        self.ditance = 0
```
Comment (FMassin) [inline apps/eew/sceewlog/eews2fcm.py:38]: `di**s**tance` ?

**Review by FMassin — CHANGES_REQUESTED** [review]
Hi @billyburgoa 

Thanks for this! But please fix the at least the error in default value of bnaFile in xml desc file, and the spelling mistake in `self.ditance` in eews2fcm.py :)

Cheers

Fred

File: apps/eew/sceewlog/config/sceewlog.cfg (hunk 1 above)
Comment (billyburgoa) [reply apps/eew/sceewlog/config/sceewlog.cfg:205]: Fred, okay but @USERCONFIGDIR@ and the other simplified way for a path are not working. That is the reason in setting a whole example path. Should it be necessary to import a SC python module?

Comment (FMassin) [reply apps/eew/sceewlog/config/sceewlog.cfg:205]: There must be a way, but we'll see that later

**Review by FMassin — APPROVED** [review]

**Pull request merged**